### PR TITLE
About me 2

### PR DESCRIPTION
**Pull request by timothy-tran**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -70,11 +78,39 @@ var myAnswer = userAnswer.toLowerCase();
...
 } else {
   alert("Sorry, you didn\'t enter right answer");
 }
+
+var numTarget = 50;
+var userInputNum;
+var myNum;
+alert("Now let\'s play a number game. You have four tries.");
+for (var i = 0; i < 4; i++) {
+  do {
```
Comment (bharding2) [inline app.js:97]: nice, a legit use-case for a do/while loop!

File: app.js [hunk 2]
```diff
@@ -70,11 +78,39 @@ var myAnswer = userAnswer.toLowerCase();
...
+    notCorrect++;
+    console.log("Your guess number:" + " " + myNum + " " + "is too low.");
+  } else if (myNum > numTarget) {
+    alert("Your guessing number is too high. Try again!");
+    notCorrect++;
+    console.log("Your guess number:" + " " + myNum + " " + "is too high.");
+  } else {
+    alert("You\'re right on target.");
+    i = 4;
+    corect++;
```
Comment (bharding2) [inline app.js:113]: you are missing an 'r' in correct

File: app.js [hunk 3]
```diff
@@ -70,11 +78,39 @@ var myAnswer = userAnswer.toLowerCase();
...
+    alert("Your guessing number is too low. Try again!");
+    notCorrect++;
+    console.log("Your guess number:" + " " + myNum + " " + "is too low.");
+  } else if (myNum > numTarget) {
+    alert("Your guessing number is too high. Try again!");
+    notCorrect++;
+    console.log("Your guess number:" + " " + myNum + " " + "is too high.");
+  } else {
+    alert("You\'re right on target.");
+    i = 4;
```
Comment (bharding2) [inline app.js:112]: this is a nice way to break the loop.  you also could have used `break;`

File: app.js (hunk 2 above)
Comment (timothy-tran) [reply app.js:113]: Thanks Ben, I think I need a new pair of glass to see it :)

**Pull request merged**